cypress/support/selectors.ts

```diff
@@ -78,20 +78,23 @@ Cypress.Commands.add('byTestDropDownMenu', (selector: string) => {
 });
 
 Cypress.Commands.add('clickNavLink', (path: [string, string?]) => {
-  cy.byTestID('nav')
-    .contains(path[0], { timeout: 10 * 1000 })
+  cy.byTestID('nav', { timeout: 15 * 1000 })
+    .should('contain', path[0])
+    .contains(path[0])
```

Comment:
```suggestion
```
`should` seems sufficient...

Reply:
once found it won't return only the HTML element containing Data Services
it will return all nav which was fed as input to should
then in the second should it will be hard to query

also not navigating - 

![image](https://user-images.githubusercontent.com/22158580/179056096-06a215c1-82d1-4d6e-b822-55d7292ae25b.png)